Source/Base/ASDisplayNode+Ancestry.mm

```diff
@@ -87,4 +87,13 @@ - (NSString *)ancestryDescription
   return strings.description;
 }
 
+- (ASDisplayNode *)firstNonLayerNode {
+  for (ASDisplayNode *node in self.supernodesIncludingSelf) {
```

Comment:
Head's up, `supernodesIncludingSelf` should not be used in hot paths because it creates a temporary NSEnumerator object. If we can use `ASFindViewOfLayer` then we should but if not, this is acceptable for now since it's in the same Ancestry header.

Reply:
Done